lambdas/service/app.py

```diff
@@ -583,6 +603,7 @@ def start_manifest_generation():
     ```
     {
         "Status": 302,
+        "Retry-After": 0,
```

Comment:
I'd prefer not to have Retry-After present so as to avoid ambiguity. 302 implies that the client doesn't have to wait.

Reply:
Also I just told @achave11 to remove it from the equivalent of this for the DSS file proxy.

Reply:
But I am open to discussion about this.

Reply:
I have the Retry-After here for consistency because the service method is returning a tuple so the view function just builds a response from that without checking the values.

Reply:
But that isn't relevant if the service returns a whole response instead of a tuple which I'll probably change to